en/docs/references/ballerina-expression-syntax.md

```diff
@@ -71,348 +71,348 @@ The boolean data type has one of the two possible values, `true` or `false`.
+### Numeric addition
+You can use the **+** operator for numeric addition. The number values can come from a literal or from a variable.
 
 #### Examples
 
-- Adding integer literals.
+- Adds integer literals:
 
   ```ballerina
   10 + 20
   ```
   
-- Adding decimal literals.
+- Adds decimal literals:
 
   ```ballerina
   0.56 + 1.84
   ```
   
-- Adding a literal and a numerical variable (`firstNumber`).
+- Adds a literal and a numerical variable (**firstNumber**):
 
   ```ballerina
   firstNumber + 100
   ```
   
-- Adding two numerical variables (`firstNumber` and `secondNumber`).
+- Adds two numerical variables (**firstNumber** and **secondNumber**):
 
   ```ballerina
   firstNumber + secondNumber
   ```
 
-### Number Subtraction
+### Numeric subtraction
 
-You can use the `-` operator for number subtraction. The number values can come from a literal or a variable.
+You can use the **-** operator for numeric subtraction. The number values can come from a literal or from a variable.
 
 #### Examples
 
-- Subtracting integer literals .
+- Subtracts integer literals:
 
   ```ballerina
   20 - 10
   ```
   
-- Subtracting decimal literals.
+- Subtracts decimal literals:
 
   ```ballerina
   1.84 - 0.56  
   ```
   
-- Subtracting a literal from a numerical variable (`firstNumber`).
+- Subtracts a literal from a numerical variable (**firstNumber**):
 
   ```ballerina
   firstNumber - 100
   ```
   
-- Subtracting a numerical variable (`secondNumber`) from another numerical variable (`firstNumber`).
+- Subtracts a numeric variable (**secondNumber**) from another numeric variable (**firstNumber**):
 
   ```ballerina
   firstNumber - secondNumber
   ```
 
-### Number Division
+### Numeric division
 
-You can use the `/` operator for number division. The number values can come from a literal or a variable.
+You can use the **/** operator for numeric division. The number values can come from a literal or from a variable.
 
 #### Examples
 
-- Dividing integer literals.
+- Divides integer literals:
 
   ```ballerina
   20 / 10 
   ```
   
-- Dividing decimal literals.
+- Divides decimal literals:
 
   ```ballerina
   1.84 / 0.56  
   ```
   
-- Dividing a numerical variable (`firstNumber`) by a literal.
+- Divides a numeric variable (**firstNumber**) by a literal:
 
   ```ballerina
   firstNumber / 100
   ```
   
-- Dividing a numerical variable (`firstNumber`) by another numerical variable (`secondNumber`).
+- Divides a numeric variable (**firstNumber**) by another numeric variable (**secondNumber**):
 
   ```ballerina
   firstNumber / secondNumber
   ```
 
-### Number Multiplication
+### Numeric multiplication
 
-You can use the `*` operator for number multiplication. The number values can come from a literal or a variable.
+You can use the ***** (asterisk) operator for numeric multiplication. The number values can come from a literal or from a variable.
```

Comment:
`*****` is not rendered properly . Lets use back ticks.

Reply:
+1